src/olympia/users/tests/test_models.py

```diff
@@ -119,6 +119,7 @@ def test_ban_and_disable_related_content(
         user.ban_and_disable_related_content()
         user.reload()
         assert user.deleted
+        assert user.banned
```

Comment:
Use `self.assertCloseToNow(user.banned)` (and below too)

Reply:
https://github.com/mozilla/addons-server/pull/12083/commits/992099fd18f0dab09512ce5faf7c5585ce50de40